docs/Release.md

```diff
@@ -49,7 +49,7 @@ This table describes all MsQuic releases, both officially supported (LTSC or SAC
 | PRE | [prerelease/1.7](https://github.com/microsoft/msquic/tree/prerelease/1.7) | N/A | Aug 13 2021 | N/A | N/A | N/A |
 | PRE | [prerelease/1.8](https://github.com/microsoft/msquic/tree/prerelease/1.8) | N/A | Sep 13 2021 | N/A | N/A | N/A |
 | PRE | [prerelease/1.9](https://github.com/microsoft/msquic/tree/prerelease/1.9) | N/A | Oct 20 2021 | N/A | N/A | N/A |
-| SAC | [release/2.0](https://github.com/microsoft/msquic/tree/release/2.0) | Windows 11 | Mar 1 2022 | May 1 2022 | Nov 1 2023 | Windows, Linux |
+| SAC | [release/2.0](https://github.com/microsoft/msquic/tree/release/2.0) | Windows 11 v22H2 | Mar 1 2022 | May 1 2022 | Nov 1 2023 | Windows, Linux |
```

Comment:
I'm not actually sure about this. I'd prefer to hold off until we've validated. Can you look at the msquic.sys file version to validate?

Reply:
it's v2.0.2.0 here
![image](https://user-images.githubusercontent.com/15126754/193126450-d8078bd2-a345-4ba3-ad7b-c0e886038e45.png)

Reply:
Ok. Cool. Thanks for validating!